config/locales/en.yml

```diff
@@ -40,3 +40,17 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    admin_deleted_work_notification:
+      subject: "[%{app_name}] Your work has been deleted by an Admin"
+      hello: "Dear %{user},"
```

Comment:
Would this be better as "greeting" or "salutation" or even "email_start_salutation"?